ovs/lib/helpers/vpool/shared.py

```diff
@@ -38,3 +40,41 @@ def retrieve_alba_arakoon_config(alba_backend_guid, ovs_client):
         if successful is False:
             raise RuntimeError('Could not load metadata from environment {0}'.format(ovs_client.ip))
         return arakoon_config
+
+    @staticmethod
+    def retrieve_local_alba_arakoon_config(alba_backend_guid, as_ini=False):
+        """
+        Retrieves the local ALBA Arakoon configuration.
+        WARNING: YOU DO NOT BELONG HERE, PLEASE MOVE TO YOUR OWN PLUGIN
+        :param alba_backend_guid: Guid of the ALBA Backend
+        :type alba_backend_guid: str
+        :return: Arakoon configuration information
+        :rtype: dict
+        """
+        cfg =  Configuration.get(REMOTE_CONFIG_BACKEND_CONFIG.format(alba_backend_guid), default=None)
+        if as_ini:
+            cfg = ArakoonClusterConfig.convert_config_to(cfg, return_type='INI')
+        return cfg
+
+    @staticmethod
+    def sync_alba_arakoon_config(alba_backend_guid, ovs_client):
+        """
+        Compares the remote and local config. Updates the local config if needed. Guarantees the latest greatest config
+        WARNING: YOU DO NOT BELONG HERE, PLEASE MOVE TO YOUR OWN PLUGIN
+        :param alba_backend_guid: Guid of the ALBA Backend
+        :type alba_backend_guid: str
+        :param ovs_client: OVS client object
+        :type ovs_client: OVSClient
+        :return: Arakoon configuration information
+        :rtype: dict
+        """
+        remote_config = VPoolShared._retrieve_remote_alba_arakoon_config(alba_backend_guid, ovs_client)
+        current_config = Configuration.get(REMOTE_CONFIG_BACKEND_CONFIG.format(alba_backend_guid), default=None)
+
+        if current_config != remote_config:
+            Configuration.set(REMOTE_CONFIG_BACKEND_CONFIG.format(alba_backend_guid), remote_config)
+
+            ini_config = ArakoonClusterConfig.convert_config_to(config=remote_config, return_type='INI')
+            Configuration.set(REMOTE_CONFIG_BACKEND_INI.format(alba_backend_guid), ini_config, raw=True)
```

Comment:
Use a sequence here. That way you guarantee that both entries are updated at the same time and both would fail in case of failures